### PR TITLE
Bump rpi-firmware to ad8608c08b122b2c228dba0ff5070d6e9519faf5 and rpi…

### PR DESCRIPTION
**Pull request by ian57**
Please make sure your PR is ready to be merged !

- [ X ] You added the changes in CHANGELOG.md
- [ X ] You choose the right repository branch to make the PR
- [ X] You described the PR as below

Changes :
- Bump rpi-firmware to ad8608c08b122b2c228dba0ff5070d6e9519faf5 and rpi-userland to a1b89e91f393c7134b4cdc36431f863bb3333163 to have the video resolution change on the fly thanks to vcgencmd commande line see https://github.com/raspberrypi/firmware/issues/637
- Useful for pixel perfect on cathodic screen
- tested on RPI3 arch

### CODE REVIEW

**Comment by digitalLumberjack** [comment]
Is this from upstream ?

If we made those changes with custom commits, the next merge with buildroot will be a mess. Best choice would be cherry picking.

File: package/rpi-firmware/config.txt [hunk 1]
```diff
@@ -1,41 +1,20 @@
...
 
-avoid_safe_mode=1
+# Disable overscan assuming the display supports displaying the full resolution
+# If the text shown on the screen disappears off the edge, comment this out
+disable_overscan=1
 
-kernel=zImage
+# How much memory in MB to assign to the GPU on Pi models having
+# 256, 512 or 1024 MB total memory
+gpu_mem_256=100
```
Comment (digitalLumberjack) [inline package/rpi-firmware/config.txt:18]: -gpu_mem_256=128
 -gpu_mem_512=256
 -gpu_mem_1024=512

**Review by digitalLumberjack — CHANGES_REQUESTED** [review]
We cannot let only 100mo for graphic memory :)

**Comment by ian57** [comment]
Ok, need to tune my PR, sorry, I thought tuning was made with patches for the config.txt.

**Comment by ian57** [comment]
Ok I will make another PR using the right method :)

**Pull request closed without merging**